src/main/webapp/app/shared/detail-overview-list/detail-overview-list.component.html

```
@@ -75,6 +75,11 @@ <h3 class="section-headline" [id]="headlinesRecord[section.headline]">{{ section
                             />
                         </dd>
                     }
+                    @case (DetailType.ProgrammingBuildPhases) {
+                        <dd id="detail-value-{{ detail.title }}">
```

HTMLHint (1.9.2)

[error] The id value [ detail-value-{{ detail.title }} ] must be unique.

(id-unique)